src/Components/WebAssembly/Build/test/BuildIntegrationTests/BuildIntegrationTest.cs

```diff
@@ -325,6 +325,29 @@ public async Task Build_WithI8NOption_CopiesI8NAssembliesWithLinkerEnabled()
             Assert.FileDoesNotExist(result, buildOutputDirectory, "wwwroot", "_framework", "_bin", "I18N.West.dll");
         }
 
+        [Fact]
+        public async Task Build_WithCustomOutputPath_Works()
+        {
+            // Arrange
+            using var project = ProjectDirectory.Create("standalone", additionalProjects: new[] { "razorclasslibrary" });
+            project.AddDirectoryBuildContent(
+@"<PropertyGroup>
+    <BaseOutputPath>$(MSBuildThisFileDirectory)build\bin\</BaseOutputPath>
+    <BaseIntermediateOutputPath>$(MSBuildThisFileDirectory)build\obj\</BaseIntermediateOutputPath>
+</PropertyGroup>");
+
+            var result = await MSBuildProcessManager.DotnetMSBuild(project, args: "/restore");
+
+            Assert.BuildPassed(result);
+
+            var compressedFilesPath = Path.Combine(
+                project.DirectoryPath,
+                "build\\obj\\Debug\\netstandard2.1",
+                "compressed");
```

Comment:
```suggestion
            var compressedFilesPath = Path.Combine(
                project.DirectoryPath,
                "build", "obj", "Debug", "netstandard2.1",
                "compressed");
```

Reply:
Oh actually we shouldn't be hardcoding `Debug` here either. I think it will be `Release` in CI. So you might need to use something like:

```
#if DEBUG
var configuration = "Debug";
#else
var configuration = "Release";
#endif
```

... to determine the value for that token.